.bazelrc

```diff
@@ -16,6 +16,7 @@ build --macos_minimum_os=10.14
 build --ios_minimum_os=12.0
 build --ios_simulator_device="iPhone 13"
 build --ios_simulator_version=15.2
+build --spawn_strategy=local
```

Comment:
Can you please add a comment with context here? Or likely even better, a link to a GitHub issue tracking re-adding sandboxing?

Reply:
Well, I expect sandboxing doesn't work because we actually try to output to local `dist/` directory. I can add a comment.